proto/cws/dumpsv1/activity_dump.proto

```diff
@@ -248,6 +256,37 @@ message BindNode {
     uint32 protocol = 5;
 }
 
+message NetworkDeviceNode {
+    repeated MatchedRule matched_rules = 1;
```

Comment:
Do we still have a use-case for sending matching rules to the backend?

Reply:
Discussed offline thanks ! 👍🏻